api/Cinematica.API/src/Cinematica.API/Models/Cognito/Users.cs

```diff
@@ -0,0 +1,8 @@
+using System.ComponentModel.DataAnnotations;
+
+public class users {
```

Comment:
Can you refer to this for mapping table names to classes? It would be better if this was called `User`, but still translated to the `users` table.
https://learn.microsoft.com/en-us/ef/core/modeling/entity-types?tabs=data-annotations#table-name